src/main/java/com/conveyal/datatools/manager/jobs/PeliasUpdateJob.java

```diff
@@ -0,0 +1,243 @@
+package com.conveyal.datatools.manager.jobs;
+
+import com.amazonaws.services.s3.AmazonS3URI;
+import com.conveyal.datatools.common.status.MonitorableJob;
+import com.conveyal.datatools.common.utils.aws.S3Utils;
+import com.conveyal.datatools.manager.auth.Auth0UserProfile;
+import com.conveyal.datatools.manager.models.Deployment;
+import com.conveyal.datatools.manager.models.FeedVersion;
+import com.conveyal.datatools.manager.persistence.Persistence;
+import com.conveyal.datatools.manager.utils.HttpUtils;
+import com.conveyal.datatools.manager.utils.json.JsonUtil;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.stream.Collectors;
+
+import static com.mongodb.client.model.Filters.in;
+
+public class PeliasUpdateJob extends MonitorableJob {
+    /**
+     * The deployment to send to Pelias
+     */
+    private Deployment deployment;
+
+    /**
+     * The workerId our request has on the webhook server. Used to get status updates
+     */
+    private String workerId;
+
+
+    /**
+     * Timer used to poll the status endpoint
+     */
+    private Timer timer;
+
+    /**
+     * Webhook authorization from username and password
+     */
+    private Header webhookAuthorization;
```

Comment:
This doesn't appear to be set anywhere.

Reply:
Yes that was a vestige from when we had webhook security. Removed in 7aab4a64041e70370cce800d09b930135cadc2d2